static_inventories/docs_library.yml

```diff
@@ -0,0 +1,15 @@
+---
+all:
+  children:
+    openstack_api:
+      hosts:
+        localhost:
+    docs:
+      hosts:
+        docs:
+          cloud_flavor: 4c4m60d
```

Comment:
That flavor name is from the Embassy cloud, but this docs sever is hosted on the Merlin cloud; it was `m1.small` previously in `group_vars` and think it should remain `m1.small` in the new YAML inventory.